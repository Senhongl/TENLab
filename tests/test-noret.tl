def foo(a, b) {
    print(a);
    print(b);
}

foo(1, 2);