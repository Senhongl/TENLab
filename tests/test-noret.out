1
[ CPUIntType{} ]
2
[ CPUIntType{} ]